server/webserver.go

```diff
@@ -805,19 +807,23 @@ func (s *LivepeerServer) StartWebserver() {
 
 			balance, err := b.BalanceAt(context.Background(), s.LivepeerNode.Eth.Account().Address, nil)
 			if err != nil {
+				glog.Error(err)
 				w.Write([]byte(""))
+			} else {
+				w.Write([]byte(balance.String()))
 			}
-			w.Write([]byte(balance.String()))
 		}
 	})
 
 	http.HandleFunc("/broadcasterDeposit", func(w http.ResponseWriter, r *http.Request) {
 		if s.LivepeerNode.Eth != nil {
 			b, err := s.LivepeerNode.Eth.BroadcasterDeposit(s.LivepeerNode.Eth.Account().Address)
 			if err != nil {
+				glog.Error(err)
 				w.Write([]byte(""))
+			} else {
+				w.Write([]byte(b.String()))
 			}
-			w.Write([]byte(b.String()))
```

Comment:
If there was an error in the call before, the returned result would still be nil instead of the empty string